cmake/modules/DuneXtCommonMacros.cmake

```diff
@@ -27,6 +27,9 @@ foreach(_boost_lib ${DS_REQUIRED_BOOST_LIBS})
   set(_BOOST_LIB "")
   string(TOUPPER "${_boost_lib}" _BOOST_LIB)
   dune_register_package_flags(LIBRARIES ${Boost_${_BOOST_LIB}_LIBRARY})
+  if (TARGET Boost::${_boost_lib})
```

Comment:
Is boost 1.70 building with cmake and exporting its targets? :open_mouth:

Reply:
Yes and ``Boost_*_LIBRARY`` is not defined anymore in that case... But I am getting weird errors from dune-testtools about a missing ``Boost::atomic`` target now, so this should not be merged yet.

Reply:
Okay, so apparently with these changes it is only working with 1.70.0 due to [this](https://gitlab.kitware.com/cmake/cmake/issues/18619). The suggested fix is to also require the boost atomic library.

Reply:
> The suggested fix is to also require the boost atomic library.

I see no problem doing that.